source/features/align-repository-header.css

```diff
@@ -0,0 +1,21 @@
+/* Align repo header to the content, just on the left side */
+.repohead > * {
+	padding-left: calc(50% - 1280px / 2) !important;
```

Comment:
This is overridden by GitHub:

![image](https://user-images.githubusercontent.com/44045911/86481681-0d510380-bd83-11ea-934e-3c3b5e023083.png)

Reply:
How’s that possible? The selector is less specific than ours

Reply:
@fregante `*` has a specificity of zero, so they are equally specific and the latter overrides the former.

Reply:
That’s weird. I’ve always thought that `>` increased specificity. 

I guess I can replace a star with a tag name and solve both issues reported in this PR

Reply:
Can't, there are 2 tag types in each level (div/nav > div/ul) and `:is()` isn't supported by Chrome yet

Reply:
It's still overridden. Can we use `.repohead > div, .repohead > nav`?